English_transcripts/school/Epic Quest/sharedAssets/C1_piant_pots_English_tutorial.txt

```diff
@@ -0,0 +1 @@
+Pick the pottery to match the picture. Drag the correct colors on to the pot to paint it. Mix colors in the mixing pot to make new colors. Can you finish painting all the pots.
```

Comment:
Can you finish painting all the pots **?**

Reply:
Updated the transcript.